website/components/LandingPage/OpenSource/index.js

```diff
@@ -117,29 +132,16 @@ function useStats() {
     cached('github', getGitHubStats).then(({ forks, stars }) => {
       setForks(formatNumber(forks));
       setStars(formatNumber(stars));
-    });
+    }, Date.now() + 2 * 60 * 1000);
```

Comment:
Only two minutes?

Reply:
This is the minimum which won't cause rate limiting set by Github. It depends on what we would like to accomplish. I choose to go with the most up to date statistics. What time interval would you suggest?

Reply:
OK, right, we've already discussed it.